### PR TITLE
Modify bake template to add skip logic if method is 'create' (task #2244)

### PR DESCRIPTION
**Pull request by georgeconstantinou**

### CODE REVIEW

File: src/Template/Bake/csv_migration.ctp [hunk 1]
```diff
@@ -28,9 +28,17 @@ class <%= $name %> extends CsvMigration
      */
     public function change()
     {
+<% if ('create' === $tableMethod): %>
```
Comment (mamchenkov) [inline src/Template/Bake/csv_migration.ctp]: @georgeconstantinou why do you need the `<% ... %>`?

Comment (georgeconstantinou) [inline src/Template/Bake/csv_migration.ctp]: This is part of the template engine, is not visible on the generated file.

**Comment by georgeconstantinou** [comment]
@mamchenkov switched to timestamp, please re-review

File: src/Shell/Task/CsvMigrationTask.php [hunk 2]
```diff
@@ -41,7 +41,7 @@ public function fileName($name)
     {
         $name = $this->getMigrationName($name);
 
-        return $this->__timestamp . '_' . Inflector::camelize($name) . $this->_getUniqueName() . '.php';
+        return $this->__timestamp . '_' . Inflector::camelize($name) . $this->__timestamp . '.php';
```
Comment (mamchenkov) [inline src/Shell/Task/CsvMigrationTask.php]: @georgeconstantinou you are using the same time stamp twice ... I meant to use the time stamp of the last modification of the `migrations.csv` file.

Comment (georgeconstantinou) [inline src/Shell/Task/CsvMigrationTask.php]: @mamchenkov please re-review.

**Pull request merged**